src/app/catalog/components/issuer-catalog/issuer-catalog.component.ts

```diff
@@ -415,6 +422,24 @@ export class IssuerCatalogComponent extends BaseRoutableComponent implements OnI
 			},
 		};
 	}
+
+	navigateToIssuer(issuerData) {
+		if (!this.loggedIn) {
+			this.router.navigate(['/public/issuers/', issuerData.slug]);
+		} else {
+			// Check whether user is a mebemr of the institution to be redirected to `issuer/issuer` page
+			this.profileManager.userProfilePromise
+				.then((profile) => profile.emails.loadedPromise)
+				.then((emails) => {
+					// Search for primary email, since it's not alawys the first in list
+					const primaryEmail = emails.entities.find((email) => email.primary).email;
+					
+					const userEmail = emails.entities[0].email;
+					const isMemeber = issuerData.staff.entities.some((staffMember) => staffMember.email === primaryEmail);
```

Comment:
could you fix the typo in ```isMember``` ? Will approve afterwards

Reply:
fixed.